backend/core/scripts/import-listings-from-csv.ts

```diff
@@ -61,7 +62,7 @@ async function main() {
         const developmentPipelineBucket: number = parseInt(
           listingFields["Development Pipeline Bucket"]
         )
-        if (projectType.toLowerCase() !== "existing occupied" && developmentPipelineBucket < 3) {
+        if (projectType?.toLowerCase() !== "existing occupied" && developmentPipelineBucket < 3) {
```

Comment:
I think the effect of this will be that if `projectType` is null/undefined, it won't step into this if-statement? (and then we'd carry on importing that listing, despite not having checked it's status wrt project type or development pipeline bucket)

My instinct would be that if we're unable to check the project type, we probably shouldn't import the listing. Maybe instead of what we have here, we could do a separate check for "is `projectType` null/undefined?".

Reply:
It should step into that if-statement. Using the `?` operator means it will early return with `undefined` if `projectType` is `undefined`, instead of calling `toLowerCase`. I verified running the script, there are a number of failures of this type.